utils/utils.h

```diff
@@ -172,7 +172,7 @@ extern struct gbConf gbConf;
               logTimeNow(timestamp, GB_TIME_STRING_BUFLEN);            \
               fprintf(fd, "[%s] %s: " fmt " [at %s+%d :<%s>]\n",       \
                       timestamp, LogLevelLookup[level],                \
-                      __VA_ARGS__, __FILE__, __LINE__, __FUNCTION__);  \
+              ##__VA_ARGS__, __FILE__, __LINE__, __FUNCTION__);        \
```

Comment:
@lxbsz 
Did we disturb this alignment ?
Or you just worried that this
##__VA_ARGS__, __FILE__, __LINE__, __FUNCTION__);\
is not okay ?

Except this, rest all looks good to me.

Reply:
@pkalever 

Fixed it. Thanks.